generator/uscore/templates/unit_tests/authorization_unit_test.rb.erb

```diff
@@ -0,0 +1,42 @@
+describe 'unauthorized search test' do
+  before do
+    @test = @sequence_class[:<%= key %>]
+    @sequence = @sequence_class.new(@instance, @client)
+<% if dynamic_search_params.present? %>
+  <% dynamic_search_params.each do |_param, search_param|  %>
+    <%= search_param[:variable_name] %> = load_json_fixture(:<%= "#{name}_#{search_param[:variable_name][1..-1]}" %>)
+      .map { |resource| FHIR.from_contents(resource.to_json) }
+    @sequence.instance_variable_set(:'<%= search_param[:variable_name] %>', <%= search_param[:variable_name] %>)
+  <% end %>
+<% end %>
+    @query = {
+      <%= search_param_string %>
+    }
+  end
+
+  it 'fails when the token refresh response has a success status' do
```

Comment:
I'm cool with printing this over again in every test, but I could see some tradeoffs here.  If we put this once in some sort of master/helper file and then pointed other tests here we'd know that all these tests were using the same code.  Plus it might save some lines of code.  For now it's not a big deal and I'll leave it to your discretion.

Reply:
I think we may want to optimize these tests for being simple and readable rather than being DRY. Since they're generated, I don't think repetition is really harmful, since we can still update a template in one place and have it update that bit of code everywhere.

I also didn't worry much about doing things optimally yet in this PR. This is the simplest of the tests to add, so after I add the second or third sets of unit tests the requirements and a potentially better structure (especially for the generator) will be more clear.